statsmodels/regression/dimred.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 from statsmodels.base import model
 import statsmodels.base.wrapper as wrap
+import pandas as pd
+import warnings
```

Comment:
Stdlib (which autocorrect thinks should be Stalin apparently), then third party, then first party, pls